### PR TITLE
Add certificate based authentication to localkube/minikube.

### PR DESCRIPTION
**Pull request by dlorenc**
This PR uses easyrsa to generate certificates on the host, and configures localkube to use them. Currently we just print out the steps to configure kubectl to use these certificates, but I'll automate this in a follow-up PR.

cc @ethernetdan @luxas

### CODE REVIEW

**Comment by dlorenc** [comment]
Addresses #10 #11

File: pkg/minikube/cluster/commands.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+var startCommand = `
+sudo killall localkube || true
+# Download and install localkube, if it doesn't exist yet.
+if [ ! -e /usr/local/bin/localkube ]; then
+	sudo curl -L %s -o /usr/local/bin/localkube
+	sudo chmod a+x /usr/local/bin/localkube;
+fi
+# Fetch easy-rsa.
+sudo mkdir -p /srv/kubernetes/certs && sudo chmod -R 777 /srv
+if [ ! -e easy-rsa.tar.gz ]; then
```
Comment (luxas) [inline pkg/minikube/cluster/commands.go]: What about creating the certs with go instead?
Maybe that's easier...

Comment (dlorenc) [inline pkg/minikube/cluster/commands.go]: I tried a few different ways, using https://github.com/SvenDowideit/generate_cert/blob/master/generate_cert.go but was unable to get anything working in pure go :(

**Comment by luxas** [comment]
What about something like https://github.com/redspread/localkube/pull/50?

**Comment by dlorenc** [comment]
> What about something like redspread/localkube#50?

I might be missing something, but that isn't quite the same. That PR doesn't setup any client certificates for actually accessing the API server, which is where I ran into trouble.

**Comment by luxas** [comment]
Ah, I was thinking about the service accounts only.
Fair enough, let's do that in another PR.

There are some small things that could be improved here, but I think that's better for follow-ups
LGTM

**Comment by luxas** [comment]
FYI, this broke `localkube` when running outside of the VM
We have to get it generated in golang :)

**Pull request merged**